### PR TITLE
black: Special formatting for ALHPAS / *_FIELDS

### PR DESCRIPTION
**Pull request by evanpurkhiser**
In response to https://github.com/kiorky/croniter/pull/116#pullrequestreview-2405799046

Part of #99

### CODE REVIEW

File: src/croniter/croniter.py [hunk 1]
```diff
@@ -75,9 +75,30 @@ def is_32bit():
...
-            "jul": 7, "aug": 8, "sep": 9, "oct": 10, "nov": 11, "dec": 12}
-DOW_ALPHAS = {"sun": 0, "mon": 1, "tue": 2, "wed": 3, "thu": 4, "fri": 5, "sat": 6}
+
+# fmt: off
+M_ALPHAS = {
+    "jan": 1, "feb": 2,  "mar": 3,  "apr": 4,  # noqa: E241
+    "may": 5, "jun": 6,  "jul": 7,  "aug": 8,  # noqa: E241
+    "sep": 9, "oct": 10, "nov": 11, "dec": 12,
+}
+DOW_ALPHAS = {
```
Comment (kiorky) [inline src/croniter/croniter.py:85]: fixup the second line if possible (use fmt:off again)

File: src/croniter/croniter.py [hunk 2]
```diff
@@ -75,9 +75,30 @@ def is_32bit():
...
+    "sun": 0, "mon": 1, "tue": 2, "wed": 3, "thu": 4,
+    "fri": 5, "sat": 6
+}
```
Comment (evanpurkhiser) [inline src/croniter/croniter.py]: Like this? @kiorky 
```suggestion
DOW_ALPHAS = {
    "sun": 0, "mon": 1, "tue": 2, "wed": 3, "thu": 4, "fri": 5, "sat": 6
}
```

Comment (kiorky) [reply src/croniter/croniter.py]: exactly.

**Comment by kiorky** [comment]
Thx again for the contribution !

**Pull request merged**